### DISCUSSION TITLE
Add support for arduino connect

### DISCUSSION THREAD

**Discussion by jellehak** [qqqlab/madflight#5] [Ideas]
Really nice project. I see you support RP2040. Thought about adding support for https://docs.arduino.cc/hardware/nano-rp2040-connect. Which uses the RP2040 but also already has an IMU integrated and wifi support.

Similar project https://github.com/RaisingAwesome/ArduinoNanoRP2040ConnectFlightController.

**Comment by qqqlab** [comment]
Thank for your feedback. Yes, the Nano RP2040 Connect is nice piece of hardware for madflight, but somebody needs to write some glue code for the IMU (LSM6DSOXTR). And, it's a pity that the LSM6DSOXTR is connected by I2C and not by the faster and more robust SPI.

It appears to me that ArduinoNanoRP2040ConnectFlightController is also fork of dRehmFlight.

**Comment by jellehak** [comment]
Yes ArduinoNanoRP2040ConnectFlightController is also a fork of dRehmFlight. The great thing about madflight is that is also adds support for various RC input protocols. I will have a look to make a HAL file for it. (currently faced the issue that arduino doesn't support printf )

**Comment by qqqlab** [comment]
The LSM6DSO(X) has been added to madflight 2.2.2
